app/src/main/res/values/rating_bar.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <style name="RatingBar" parent="Theme.AppCompat">
```

Comment:
does this style look the same on all version of android? try api 21, 30 and something in the middle